src/components/Button.astro

```diff
@@ -34,4 +34,19 @@ const { text, id } = Astro.props;
     width: var(--lh);
     height: var(--lh);
   }
+
```

Comment:
This one and all other components, do not require any changes. Replacing custom properties `--color-bg` and `--color-bg-contrast` should control colour changes without making any changes here (and all other components).